great_expectations/expectations/metrics/query_metrics/query_column_pair.py

```diff
@@ -36,7 +36,7 @@ def _sqlalchemy(
         metric_value_kwargs: dict,
         metrics: Dict[str, Any],
         runtime_configuration: dict,
-    ) -> List[sqlalchemy_engine_Row]:
+    ) -> List[dict]:
```

Comment:
What is the reason for this change?

Reply:
@Shinnnyshinshin This is also very important.  The job of the metric is to return backend-agnostic output, regardless of the backend.  In the map cases, the decorator will take care of it, but here we are dealing with a raw metric, so it must take care of it.

Reply:
Do you anything about the structure of the `dict`? Are the keys `str`s or is this the result of processing data that can come from anywhere and we don't know the structure?